src/main/resources/site/parts/frontPageBanner/frontPageBanner.html

```diff
@@ -2,7 +2,7 @@
     <div class="container">
         <div class="row">
             <div class="col-12 col-lg-6 mx-auto">
```

Comment:
Just a suggestion; maybe add a little padding/margin at the bottom?

Reply:
fixd